graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/method/MethodDataFetcherFactory.kt

```diff
@@ -39,16 +43,31 @@ class MethodDataFetcherFactory(
     private val resolvers = ArgumentResolverComposite(argumentResolvers)
 
     fun createDataFetcher(bean: Any, method: Method): DataFetcher<Any?> {
+        if (isTrivial(method) || isTrivial(bean.javaClass)) {
```

Comment:
Wondering if we should allow this on class level. It's probably unlikely that it's adding a lot of convenience, while it would be easy to have an unintended side-effect on datafetchers added later to an existing class.

Reply:
I can just remove it for now then.